raiden/tests/unit/test_transfer.py

```diff
@@ -158,16 +157,17 @@ def test_transfer(raiden_network):
     assert isinstance(a1_recv_messages[0], DirectTransfer)
 
 
+@pytest.mark.timeout(10)
```

Comment:
Yes I understand that but why add it? Are we going to start adding small timeouts to whichever tests do not use a geth node now?